### DISCUSSION TITLE
Genome version?

### DISCUSSION THREAD

**Discussion by mkyriak** [cccnrc/plot-VCF#5] [General]
Hello, I am wondering if I can change the version of the genome, so instead of hg38, can I use b37 or hg19?

Thank you!

**Comment by cccnrc** [comment]
Sure thing, you can both specify `hg19` or `GRCh37` in the `ASSEMBLY=` option. It works with all functions. 

You can also use your own [Fasta file](https://github.com/cccnrc/plot-VCF#fasta-file) in case you want to analyze other non-hg38/hg19 assembly.

Please note, most of the debugging and usage has been performed in `hg38`, if you find any bug etc. just open issues, I will be glad to correct them!
